From 6001963e5c3678623e843b9f5bde2ac3e615f707 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Wed, 21 Sep 2016 01:02:00 +0200
Subject: [PATCH 4/8] Disable dumping schema.rb after db:migrate

db/schema.rb is owned by root, so not writable for git user.
---
 config/environments/production.rb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/environments/production.rb b/config/environments/production.rb
index a9d8ac4..05377f7 100644
--- a/config/environments/production.rb
+++ b/config/environments/production.rb
@@ -80,4 +80,6 @@ Rails.application.configure do
   config.eager_load = true
 
   config.allow_concurrency = false
+
+  config.active_record.dump_schema_after_migration = false
 end
-- 
2.10.1 (Apple Git-78)

